app/controllers/api/checkout_controller.rb

```diff
@@ -133,4 +124,12 @@ def authenticate_checkout
       nil
     end
   end
+
+  def authenticate_card
+    @uuid = params[:uuid]
+    @card = CheckoutCard.find_by(uuid: @uuid)
+    render status: :unauthorized, json: I18n.t('checkout.error.not_activated') unless @card.active
```

Comment:
Should check if @card is nil before this now it happens afterwards in 'info' or never